lib/OpenQA/Schema/Result/JobGroups.pm

```diff
@@ -146,6 +146,22 @@ sub rendered_description {
     return;
 }
 
+sub full_name {
+    my ($self) = @_;
+
+    if (my $parent = $self->parent) {
+        return $parent->name . ' / ' . $self->name;
+    }
+    else {
+        return $self->name;
+    }
+}
```

Comment:
personal taste: How about

```perl
return $self->name unless $self->parent;
return $self->parent->name . ' / ' . $self->name;
```

Reply:
i changed it to

```
    if (my $parent = $self->parent) {
        return $parent->name . ' / ' . $self->name;
    }
    return $self->name;
```

This is short/readable enough and still saves querying the parent twice.